website/project/model.py

```diff
@@ -529,6 +524,8 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     #: Whether this is a pointer or not
     primary = True
 
+    is_draft_registration = False
```

Comment:
These methods were added in an earlier PR. If you have questions or concerns about code that is already part of our codebase, then I feel that discussion should not be part of the review of an unrelated PR. 

At any rate, @HarryRybacki this line can be deleted as it is unused.